ui/webui/src/components/app.scss

```diff
@@ -15,3 +15,10 @@ html:not(.index-page) body {
 #installation-wizard .pf-c-wizard__main-body {
   flex: 1 1 auto;
 }
+
+// workaround for a LogViewer bug
+// - without this the line number column is too narrow
+//   and line numbers quickly become unreadable
+.pf-c-log-viewer__index {
+  width: auto;
```

Comment:
Doesn't this make the UI jump if new numbers are added and are longer? Eg. 99->100.

Perhaps also `min-width: 3rem;` or some such could be useful to prevent the log from jumping too much in the beginning when the render length of line numbers grows faster.

Reply:
I wouldn't mess with it more that we do now. It is a workaround for a bug in Patternfly anyway.